test/common_utils.py

```diff
@@ -72,10 +73,10 @@ def add_batch_choices(a):
             yield pytree.tree_unflatten(batched_args, arg_spec), pytree.tree_unflatten(in_dims, arg_spec), kwarg_values
 
 
-def get_fallback_and_vmap_exhaustive(op, arg_values, kwarg_values, compute_loop_out=True):
+def get_fallback_and_vmap_exhaustive(op, arg_values, kwarg_values, compute_loop_out=True, first_dim_only=False):
```

Comment:
ditto here: this reads better as `get_fallback_and_vmap_exhaustive(op, arg_values, kwarg_values, bdims=(0, 1))`